### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by Mattmatt10111**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -22,6 +22,16 @@
...
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Dead Island 2/Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Mattmatt10111/asl-scripts/master/DeadIsland2.asl</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Autosplitter and Load Removal by Mattmatt></Description>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:33]:
```suggestion
        <Description>Autosplitter and Load Removal by Mattmatt</Description>
```

**Comment by Mattmatt10111** [comment]
Why did the checks fail?

**Comment by just-ero** [comment]
cc @wooferzfg

**Comment by Mattmatt10111** [comment]
Should I make a new pull request or will that likely also fail the checks again?

**Pull request closed without merging**